scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -201,9 +215,21 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     case Tokenized.Error(_, _, details) => throw details
   }
   lazy val (parserTokens, parserTokenPositions) = {
+    def indentedRegion(regions: List[SepRegion]): Boolean =
+      regions.headOption.exists {
+        case SepRegion(c, _, _) => c == RegionIndent || c == RegionIndentEnum
+      }
+
     val parserTokens = mutable.ArrayBuilder.make[Token]
     val parserTokenPositions = mutable.ArrayBuilder.make[Int]
-    @tailrec def loop(prevPos: Int, currPos: Int, sepRegions: List[Char]): Unit = {
+
+    @tailrec def loop(
+        prevPos: Int,
+        currPos: Int,
+        sepRegionsParameter: List[SepRegion],
```

Comment:
This is the default so maybe let's leave out `Parameter`
```suggestion
        sepRegions: List[SepRegion],
```

Reply:
it would clash with sepRegions defined below